openarena-q3e

```diff
@@ -1,29 +1,66 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
+# Check if the config file exists
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+# Check if it's the first run or if RESPECT_XDGDIRS is not set
+if [ ! -f "$CONFIG_FILE" ] || [ -z "$RESPECT_XDGDIRS" ]; then
+    RESPECT_XDGDIRS=0
+
+    # Save RESPECT_XDGDIRS in the config file
+    echo "RESPECT_XDGDIRS=$RESPECT_XDGDIRS" > "$CONFIG_FILE"
+    echo -e "It seems this is the first time you run OpenArena... You can edit the $$CONFIG_FILE and change RESPECT_XDGDIRS to use your XDG_DATA_FOLDER, which is: ${XDG_DATA_HOME}/homepath Instead of polluting your HOMEDIR by adding the hidden .openarena folder. \n"
+fi
+# Use the value of RESPECT_XDGDIRS to determine the directory
+if [ -n "$RESPECT_XDGDIRS" ] && [ "$RESPECT_XDGDIRS" -eq 1 ]; then
+    # Use ${XDG_DATA_HOME}/homepath directory
+    mkdir -p "${XDG_DATA_HOME}/homepath"
+    OA_DIR="${XDG_DATA_HOME}/homepath"
+    if [ -d "$HOME/.openarena" ]; then
+        mv "$HOME/.openarena"/* "$OA_DIR"
+        rm -rf "$HOME/.openarena"
+    fi
+    if [ -d "$HOME/.var/app/io.github.ec_.Quake3e.OpenArena/.openarena" ]; then
```

Comment:
This seems to be incorrect. `$HOME/.openarena` should probably be used here (and on the next line).

Reply:
No, that's to remove the duplicate made by the flatpak manifest.